### ISSUE TITLE
DEPLOYING CONFIGURATION M365DSC

### ISSUE THREAD

**Issue by opesy** [microsoft/Microsoft365DSC#3287]
I am getting this issue when deploying to my target tenant. This is a brand new test tenant, I have granted the user account permision using Update-M365DSCAllowedGraphScopes -ALL -Type Update.
 I do not know what maybe wrong again

![Screenshot 2023-05-10 152905](https://github.com/microsoft/Microsoft365DSC/assets/90148829/5327a1c8-a898-46ad-aeaf-82b7b8abbab1)

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
# insert configuration here
```

**Comment by andikrueger** [comment]
Please run the two commands listed in the last line of the error message. This should fix your system.

**Comment by andikrueger** [comment]
Please run Get-Module -ListAvailable and share the outcome.

**Comment by opesy** [comment]
> Please run Get-Module -ListAvailable and share the outcome.

![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/dffd85ce-1f34-4a83-8026-10412b60e000)
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/d8444bd8-c9ce-47ee-b6fa-4ef1a8be5eeb)
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/161d2759-b7af-46c6-80dc-37cdda8ac767)

**Comment by andikrueger** [comment]
The modules are installed at the wrong location. For DSC they need to be installed in the global modules folder. Right now they are within your PS Profile.

**Comment by opesy** [comment]
> The modules are installed at the wrong location. For DSC they need to be installed in the global modules folder. Right now they are within your PS Profile.

okay, can you advice how i can correct this? my PS proficiency isn't strong

**Comment by andikrueger** [comment]
Please uninstall all the modules that are listed in this file: https://github.com/microsoft/Microsoft365DSC/blob/Dev/Modules/Microsoft365DSC/Dependencies/Manifest.psd1

**Comment by opesy** [comment]
uninstall completed

**Comment by opesy** [comment]
> Get-Module -ListAvailable

Hi, It seems to have ran completely but i am getting some errors like this at some intervals
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/e41a0b0e-af40-4e29-b3bb-5a40b6e886c2)


how long will it take to see replications in my target tenant. I have only seen some groups created and mailbox flows but nothing on Intune, Security ,Compliance etc

**Comment by andikrueger** [comment]
Please review the event log of under m365dsc using event viewer. There should be additional information about the issues.

**Comment by opesy** [comment]
I saw some errors and this is the comments on them, quite repetitive. What do they mean
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/f3e492ad-6e12-4a9f-94ac-aa64206512be)
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/afb26449-01ec-4d98-8e04-eb50d90e480f)
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/7a35afec-ef4c-41e7-adf9-ca98029ee222)

**Comment by andikrueger** [comment]
Could you share a screen shot for the App Registration Page and the granted permissions for the app you are using? The messages above show a lack of permissions granted.

https://microsoft365dsc.com/user-guide/get-started/authentication-and-permissions/ Please review this guide to grant the right permissions.

**Comment by opesy** [comment]
I am not using an enterprise app, I am authenticating via username and password as this allows me to clone all workloads

**Comment by andikrueger** [comment]
... and still you need an enterprise app to allow the access to the data within the tenant. Please use this guide: https://microsoft365dsc.com/user-guide/get-started/introduction/

**Comment by opesy** [comment]
Hi, I have created and granted the Graph command line tools and Enterprise application the devicemanagementconfig/manageddevices permissions. but i still get this error
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/ad21dd2a-21bc-4918-8374-49914a45269a)
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/17f4cb4f-7271-4588-930f-15c524370403)

other workloads deployed on the tenant using credentials except Intune  so I am using application secret for intune and this is what is giving me the errors

**Comment by opesy** [comment]
when i use credential for intune, this is what happens;
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/867e48be-45b6-48a6-95d1-eefcb4941e60)

**Comment by andikrueger** [comment]
Do you have the appropriate rights for Intune?

**Comment by opesy** [comment]
yes, i do...so I was a able to figure out the intune using application id and Hash but i noticed that not all profiles/settings where imported to my new tenant. what may cause this

I also observed that I got this error at the end of deploying SC to my new tenant
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/575ac06f-59f2-4738-bc6f-791dc65d85f5)

got this error for OneDrive
![image](https://github.com/microsoft/Microsoft365DSC/assets/90148829/f10d3c93-7a14-4a53-bedd-afbd663f20bb)

I ran multiple workloads on different PS at the same time, can this be an issue. (deploying different work load at the same time to my new tenant. )
